modules/updateChecker.js

```diff
@@ -0,0 +1,115 @@
+"use strict";
+
+const _ = global._;
+const got = require('got');
+const semver = require('semver');
+const Windows = require('./windows');
+const log = require('./utils/logger').create('updateChecker');
+
+
+
+/**
+ * Check for updates to the app.
+ * @return {[type]} [description]
+ */
+const check = exports.check = function() {
+    log.info('Check for update...');
+
+    let str = null;
+
+    switch (global.mode) {
+        case 'mist':
+            str = 'mist';
+            break;
+        case 'wallet':
+            str = 'wallet';
+            break;
+    }
+
+    return got('https://api.github.com/repos/ethereum/mist/releases', {
```

Comment:
`fetch` isn't yet available in Node, it seems. So I opted for a lightweight wrapper around Node's `request`.